### ISSUE TITLE
Comments on draft

### ISSUE THREAD

**Issue by rjleveque** [amath574w2015/am574-group03#2]
I'm just starting a thread for comments on your draft -- I'll try to add some when I get a chance to read it in more detail, other groups please add comments below too.

**Comment by rochekj** [comment]
Hi Randy, I may have been over zealous on the number of examples and may backtrack on that a little. the draft is certainly a work in progress still so please be aware of that. all this to say, your guidance is absolutely welcome.

**Comment by rjleveque** [comment]
There's a figure need to latex your draft that's missing from the repo: `berg-lev-fig21`.

It's fine if you want to just concentrate on one system (or scalar) equation, rather than trying to tackle advection, Burgers', acoustics, and shallow water.

**Comment by rochekj** [comment]
hi randy, I uploaded the figure and modified the text considerably. we are converging on paper content as we realize our limits with the implementations.
we will likely downsize the number of examples as you noted.

**Comment by rjleveque** [comment]
I notice `paper_draft1.aux` is in the repository -- did you instead mean to commit `paper_draft1.pdf`?

**Comment by rochekj** [comment]
Yes, am careless. Current status is there now, but we are still refining (pun kind of intended).
Also, we will send a note probably Sunday night for you to look at our draft. It is coming together now. We will cut some of the examples to make space for implementation comments and results.
